examples/ui-demo/src/components/preview/AuthCardWrapper.tsx

```diff
@@ -14,6 +14,10 @@ export function AuthCardWrapper({ className }: { className?: string }) {
       output.push([{ type: "email" as const }]);
     }
 
+    if (config.auth.showExternalWallets) {
+      output.push([{ type: 'injected' as const }])
```

Comment:
how do you think we should extend this so that users can specify which wallets to show? I imagine the options in the UI will have to be something like `Injected, Wallet Connect, Coinbase, Safe` and users can choose to show like just injected and maybe coinbase wallet or something